sabnzbd/filesystem.py

```diff
@@ -792,23 +792,25 @@ def find_dir(p):
     # windows diskfree
     try:
         # Careful here, because win32api test hasn't been done yet!
-        import win32api
-    except:
-        pass
+        import win32api as w32
+    except ImportError:
+        w32 = None
```

Comment:
Interesting approach, is this the recommended way?

Reply:
I'm still trying to figure out the recommended way.  Normally an ImportError that isn't dealt with ends the whole script.  Since this keeps going I need to define the variable and catch it when the module isn't there.  

I changed the name just out of habit in case win32api was being called elsewhere to avoid clobbering it.  I mean, it shouldn't be called elsewhere, but I'm still working my way through this codebase and am very unfamiliar with it.  Until I'm more confident I understand it I was playing it safe.  Just like the "T" confusion I caused earlier!